FCCee/CLD/compact/CLD_o3_v01/InnerTracker_o2_v07.xml

```diff
@@ -38,34 +38,47 @@
         <detector name="InnerTrackers" type="DD4hep_SubdetectorAssembly" vis="ITVis">
 
             <shape type="Polycone" material="Air">
-              <zplane z="-InnerTracker_half_length"       rmin="ITEnvelopeClearanceConeHalfAngle * ConeBeamPipe_zmax"             rmax="InnerTracker_outer_radius" />
-              <zplane z="-(Vertex_half_length + 5 * mm)"  rmin="ITEnvelopeClearanceConeHalfAngle * (Vertex_half_length + 5 * mm)" rmax="InnerTracker_outer_radius" />
-              <zplane z="-(Vertex_half_length)"           rmin="Vertex_outer_radius"                                              rmax="InnerTracker_outer_radius" />
-              <zplane z="+(Vertex_half_length)"           rmin="Vertex_outer_radius"                                              rmax="InnerTracker_outer_radius" />
-              <zplane z="+(Vertex_half_length + 5 * mm)"  rmin="ITEnvelopeClearanceConeHalfAngle * (Vertex_half_length + 5 * mm)" rmax="InnerTracker_outer_radius" />
-              <zplane z="+InnerTracker_half_length"       rmin="ITEnvelopeClearanceConeHalfAngle * ConeBeamPipe_zmax"             rmax="InnerTracker_outer_radius" />
-	    </shape>
+                <zplane z="-InnerTracker_half_length"
```

Comment:
Is this just white-space change?

Reply:
it seems so, i will fix together with the change in size of the inner tracker